src/main/java/ch/heigvd/dai/Main.java

```diff
@@ -1,12 +1,26 @@
 package ch.heigvd.dai;
 
+import ch.heigvd.dai.commands.Root;
 import java.io.File;
 import picocli.CommandLine;
 
 public class Main {
 
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        // Define command name - source: https://stackoverflow.com/a/11159435

```

Comment:
Since the Root classe is relatively small, what about having only the `Main` class ?

Reply:
It seems that we can do it that way:
```java
class OutputGroup {
    @Option(names = "--output", description = "Output file")
    File output;

    @Option(names = "--output-format", description = "Output format")
    String outputFormat;
}
```
```java
@Command
class MyCommand {
    @ArgGroup(exclusive = false)
    OutputGroup outputGroup;

    // other options and command logic
}
```

```java
@Command(validateWith = OutputGroupValidator.class)
class MyCommand {
    // ...
}

class OutputGroupValidator implements IParameterValidator2 {
    public void validate(CommandLine commandLine) throws ParameterException {
        OutputGroup group = commandLine.getCommandSpec().userObject().outputGroup;
        if (group != null && group.outputFormat != null && group.output == null) {
            throw new ParameterException(commandLine, 
                "If --output-format is specified, --output must also be specified");
        }
    }
}
```

Reply:
I believe this may not be the best approach. Currently, we only have one root command; however, if we plan to implement additional subcommands, such as decrypting QR codes or other functionalities, it would be advisable to retain the command folder to accommodate these future enhancements.